### PR TITLE
Noetic tweaks

### PR DESCRIPTION
**Pull request by erelson**
Minor fixups to get things building/working on 20.04.

Also forward-ports #150

### CODE REVIEW

File: fetch_teleop/scripts/tuck_arm.py [hunk 1]
```diff
@@ -73,8 +76,10 @@ def __init__(self):
     def run(self):
         move_thread = None
         if not is_moveit_running():
-            rospy.loginfo("starting moveit")
+            rospy.loginfo("Starting MoveIt...")
             move_thread = MoveItThread()
+        else:
+            prospy.loginfo("MoveIt already started...")
```
Comment (mikeferguson) [inline fetch_teleop/scripts/tuck_arm.py]: Pretty sure this is supposed to be "rospy" not "prospy"

**Review by cjds — APPROVED** [review]
Apart from prospy lgtm

File: fetch_teleop/scripts/tuck_arm.py (hunk 1 above)
Comment (erelson) [reply fetch_teleop/scripts/tuck_arm.py]: 🤦 thanks!

**Review by JeffWilson7 — APPROVED** [review]
lgtm

But heads up that arm tuck relies on roslaunch respawn in melodic, might have to make that change for noetic:
https://github.com/fetchrobotics/fetch_robots/pull/59

**Comment by erelson** [comment]
Good catch. Tacked that fix onto this PR for noetic: https://github.com/fetchrobotics/fetch_robots/pull/61

**Pull request merged**